app.js

```diff
@@ -20,24 +21,23 @@ app.get("/api/articles/:article_id", getArticle);
 
 app.get("/api/articles/:article_id/comments", getArticleComments);
 
-app.post("/api/articles/:article_id/comments", postArticleComment)
+app.post("/api/articles/:article_id/comments", postArticleComment);
 
-app.use((err, req, res, next) => {
-
-  if (err.status === 404) {
-    res.status(404).send({ msg: "Article id is not found" });
-  }
-  next(err);
-});
+app.patch("/api/articles/:article_id", patchArticle),
+  app.use((err, req, res, next) => {
+    if (err.status === 404) {
+      res.status(404).send({ msg: "Article id is not found" });
+    }
+    next(err);
+  });
 
 app.use((err, req, res, next) => {
   if (err.code) {
-    res.status(400).send({ msg: "Input does not exist" });
+    res.status(400).send({ msg: "Not a valid input" });
```

Comment:
Please specify the err.code e.g. 22P02 as you can send custom errors based on the codes.